examples/amm/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "amm"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
+edition = "2021"
+
+[dependencies]
+ink_primitives = { version = "4.0.0-alpha.1", default-features = false }
+ink_metadata = { version = "4.0.0-alpha.1", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "4.0.0-alpha.1", default-features = false }
+ink_storage = { version = "4.0.0-alpha.1", default-features = false }
+ink_lang = { version = "4.0.0-alpha.1", default-features = false }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+
+[lib]
+name = "amm"
+path = "lib.rs"
+crate-type = [
+	"cdylib",
+]
+
+# This is necessary to get things to compile. Without it the compiler throws an error. 
+# Setting overflow checks to "false" for an AMM is typically not a good idea...
+# "This typically goes back to a known bug in the Rust compiler:
+# https://github.com/rust-lang/rust/issues/78744"
+[profile.release.package.compiler_builtins]
+overflow-checks = false
```

Comment:
I thought we fixed this ages ago, are you sure it's still needed?

Reply:
I get this error without it:

```sh
ERROR: Validation of the Wasm failed.


ERROR: An unexpected panic function import was found in the contract Wasm.
This typically goes back to a known bug in the Rust compiler:
https://github.com/rust-lang/rust/issues/78744

As a workaround try to insert `overflow-checks = false` into your `Cargo.toml`.
This will disable safe math operations, but unfortunately we are currently not
aware of a better workaround until the bug in the compiler is fixed.


ERROR: An unexpected import function was found in the contract Wasm: _ZN4core9panicking5panic17hca097ff786771598E.
The only allowed import functions are those starting with one of the following prefixes:
seal, memory
```